src/components/CubeOverviewModal.js

```diff
@@ -26,14 +26,17 @@ class CubeOverviewModal extends Component {
   constructor(props) {
     super(props);
 
+    props.cube.descriptionhtml= props.cube.descriptionhtml && props.cube.descriptionhtml !== 'undefined' ? props.cube.descriptionhtml : props.cube.description;
```

Comment:
This totally breaks React's data model and you should not do this (meaning modifying incoming props).